forge-gui/res/cardsfolder/upcoming/obscura_interceptor.txt

```diff
@@ -0,0 +1,12 @@
+Name:Obscura Interceptor
+ManaCost:1 W U B
+Types:Creature Cephalid Wizard
+PT:3/1
+K:Flash
+K:Lifelink
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigConnive | TriggerDescription$ When CARDNAME enters the battlefield, it connives. When it connives this way, return up to one target spell to its owner's hand. (To have a creature connive, draw a card, then discard a card. If you discarded a nonland card, put a +1/+1 counter on that creature.)
+SVar:TrigConnive:DB$ Connive | SubAbility$ DBImmediateTrigger
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | Execute$ TrigReturn | TriggerDescription$ When it connives this way, return up to one target spell to its owner's hand.
```

Comment:
Wasn't sure how else to make this work... we may just need more rules... if Connive can't fail in some way, then this seems fine?

I had a draft for ConnivesTrigger all ready, too, but I'm not sure this really needs it?

Reply:
depending on the Rules, it might be if it is still on the battlefield?

otherwise "When it connives this way" wouldn't make much sense?

Reply:
Seems a logical enough check – do you think we should add a check that it's still on the battlefield or just leave these as is?

Either way we'll definitely need revisit these once we get rules

Reply:
I guess that settles it?